### PR TITLE
Convert config, properties, dependencies, and migrations from docker to JS

### PR DESCRIPTION
**Pull request by cryptodread**
~~- [x] Update to 2.2.9~~ already done in https://github.com/Start9Labs/sphinx-relay-wrapper/pull/15
- [x] Add JS improvements
- [x] LND v15 compatibility

### CODE REVIEW

File: scripts/services/migrations.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        ),
+        down: compat.migrations.updateConfig(
+          (config) => {
+            return config;
+          },
+          true,
+          { version: "2.2.7", type: "down" },
+        ),
+      },
+    },
```
Comment (chrisguida) [inline scripts/services/migrations.ts:22]: Pretty sure you don't need to define null migrations @dr-bonez

Comment (cryptodread) [reply scripts/services/migrations.ts:22]: so you are suggesting I remove the migrations.ts file?

Comment (chrisguida) [reply scripts/services/migrations.ts:22]: Good point, @dr-bonez how should we define a migration if we want to skip needs config always?

Comment (cryptodread) [reply scripts/services/migrations.ts]: code removed, left the migrations.ts file in place for future migration conditions.

**Comment by chrisguida** [comment]
To clarify, this PR doesn't update to 2.2.9, that work is already on master.

What this PR *does* do is convert all docker procedures to js procedures, except health checks and backups. So: config, properties, dependencies, and migrations

File: scripts/services/migrations.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { compat, types as T } from "../deps.ts";
+
+export const migration: T.ExpectedExports.migration = compat.migrations
+  .fromMapping( {}, "2.2.9" );
```
Comment (chrisguida) [inline scripts/services/migrations.ts:4]: Looks good

**Review by chrisguida — APPROVED** [review]
Looks good to me!

**Pull request merged**